apps/dotcms-ui/src/app/portlets/dot-graphql/dot-graphql.component.scss

```diff
@@ -0,0 +1,15 @@
+@use "variables" as *;
+
+:host {
```

Comment:
We have components for it: https://dotcms.github.io/core-web/?path=/story/dotcms-portlet--no-actions